lib/cocina_checker.rb

```diff
@@ -7,9 +7,12 @@ def self.check
   end
 
   def check_cocina
-    puts "repos to Cocina check: #{repos.map(&:name).join(', ')}"
-
-    version_map = Dir["#{base_directory}**/Gemfile.lock"].filter_map do |lockfile_path|
+    gem_filename = 'Gemfile.lock'
```

Comment:
Move to a constant?

Reply:
How about the settings.yml file for consistency with other fixed values?